templates/daemon.upstart.erb

```diff
@@ -0,0 +1,30 @@
+# Prometheus <%= @name %> Exporter (Upstart unit)
+description "Prometheus exporter <%= @name %>"
+start on runlevel [2345]
+stop on runlevel [06]
+
+env EXPORTER=<%= @bin_dir %>/<%= @name %>
+env USER=<%= @user %>
+env GROUP=<%= @group %>
+env DEFAULTS=/etc/default/<%= @name %>
+env RUNDIR=/var/run/<%= @name %>
+env PID_FILE=/var/run/<%= @name %>/<%= @name %>.pid
```

Comment:
This would be simpler as `env PID_FILE=$RUNDIR/<%= @name %>.pid` but I'm not entirely sure if that works.